core-libs/setup/ssr/error-handling/server-error-response/propagate-server-error-response.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InjectionToken } from '@angular/core';
+import { CxServerErrorResponse } from './cx-server-error-response';
+
+/**
+ * Injection token used to provide a function that will propagate the server error response occurred during server side rendering to the CommonEngine wrapper.
```

Comment:
1. 
```suggestion
 * Function that will propagate the server error response object
 * created during server side rendering to the CommonEngine wrapper.
```

2. `CommonEngine wrapper` vs `CxCommonEngine`?